docs/source/datasheet/simple_registers/index.rst

```diff
@@ -7,3 +7,4 @@ Simple Registers
    :maxdepth: 2
 
    blinking
+   Clock Divider
```

Comment:
Should be the same as file name, i.e., clk_divider
Otherwise, the new page will not be included

Reply:
Updated.